# Exam 1 Answers Miguel Rey Romero

# Problem 2

In [1]:
ls -lh

total 2.4M
-rw-rw-r-- 1 osboxes osboxes 1.2M Sep  6 09:52  conv.txt
-rw-rw-r-- 1 osboxes osboxes  147 Sep  6 09:52  DISCLAIMER
-rw-rw-r-- 1 osboxes osboxes    0 Sep  6 11:07  Exam_1_Answer
-rw-rw-r-- 1 osboxes osboxes 1.4K Sep  6 11:20  Exam_1_Answers.ipynb
-rw-rw-r-- 1 osboxes osboxes 5.5K Sep  6 09:52 'Exam Week 1.ipynb'
-rw-rw-r-- 1 osboxes osboxes 3.9K Sep  6 09:52 'Exam Week 2.ipynb'
-rw-rw-r-- 1 osboxes osboxes 4.2K Sep  6 09:52  Germplasm.tsv
-rw-rw-r-- 1 osboxes osboxes  141 Sep  6 09:52  how_to_convert.txt
-rw-rw-r-- 1 osboxes osboxes 1.1K Sep  6 09:52  LICENSE
-rw-rw-r-- 1 osboxes osboxes  637 Sep  6 09:52  LocusGene.tsv
-rw-rw-r-- 1 osboxes osboxes 1.2M Sep  6 09:52  Locus_Germplasm_Phenotype_20130122.txt
-rw-rw-r-- 1 osboxes osboxes  743 Sep  6 09:52  README.md


### The permissions of that file are: 
#### "rw" (read and write) to users.
#### "rw" (read and write) to groups.
#### "r" (read) to anyone.

# Problem 3

In [5]:
head -1 Locus_Germplasm_Phenotype_20130122.txt

Locus_name)	Germplasm_name	phenotype	pubmed_id


### The command head prints first 10 lines, the argument -1 prints only the first line.

# Problem 4

In [6]:
wc -l Locus_Germplasm_Phenotype_20130122.txt

7216 Locus_Germplasm_Phenotype_20130122.txt


The command wc (word count) allows for different counts, the argument -l count the lines. The Locus_Germplasm_Phenotype_20130122.txt have 7216 lines.